images/protected-publish/TESTING.txt

```diff
@@ -0,0 +1,92 @@
+There is some curated testing data located in:
```

Comment:
This file doesn't have to be part of it, it's just here in case it's helpful during evaluation of the PR.

Reply:
Or it could be fixed up however is needed and incorporated into the README.